go/vt/schemadiff/diff_test.go

```diff
@@ -667,6 +667,7 @@ func TestDiffSchemas(t *testing.T) {
 					// validate schema1 unaffected by Apply
 					assert.Equal(t, schema1SQL, schema1.ToSQL())
 
+					require.NotNil(t, schema2)
```

Comment:
Is this check useful? We already check earlier that there's no error returned when `schema2` is created?

Reply:
removed